src/test/java/com/scottlogic/AuthorPostSorterTest.java

```diff
@@ -12,75 +12,70 @@
 
 class AuthorPostSorterTest {
 
+    UserPost userPost1 = new UserPost("Joe Bloggs",
+            OffsetDateTime.of(2020, 1, 3, 7, 12, 3, 0, ZoneOffset.UTC),
+            "Hello World!", 2);
+
+    UserPost userPost2 = new UserPost("Joe Bloggs",
+            OffsetDateTime.of(2020, 1, 3, 7, 12, 3, 0, ZoneOffset.UTC),
+            "Hello World!", 2);
+
+    UserPost userPost3 = new UserPost("Albert Einstein",
+            OffsetDateTime.of(2020, 1, 3, 8, 53, 34, 0, ZoneOffset.UTC),
+            "Another example post.", 1);
+
+    UserPost userPost4 = new UserPost("Cucumber",
+            OffsetDateTime.of(2021, 3, 12, 13, 22, 12, 0, ZoneOffset.UTC),
+            "An example of a post \nwith lines breaks.", 3);
+
+    UserPost userPost5 = new UserPost("Alreen Trapezoid",
+            OffsetDateTime.of(2020, 1, 3, 7, 12, 3, 0, ZoneOffset.UTC),
+            "Hello World!", 2);
+
     @Test
-    void sort_withNull_returnsNull() {
+    void authorPostSort_withNull_returnsNull() {
         List<UserPost> actual = null;
         actual = new AuthorPostSorter().sort(actual, SortOrder.ASC);
         Assertions.assertEquals(null, actual);
```

Comment:
I think it's best not to have the actual test overwrite your initial test data
```suggestion
        List<UserPost> initialList = null;
        List<UserPost> sortedList = new AuthorPostSorter().sort(initialList, SortOrder.ASC);
        Assertions.assertEquals(null, sortedList);
```